django-stubs/contrib/sessions/base_session.pyi

```diff
@@ -16,6 +16,11 @@ class AbstractBaseSession(models.Model):
     session_key: str
     objects: Any
 
+    class Meta:
+        verbose_name: str
+        verbose_name_plural: str
+        abstract: bool
```

Comment:
What about `Literal[True]`?

Reply:
I think using `Literal[True]` here won't help anyone, to the contrary.

If the user subclasses this Meta and sets `abstract = False`, mypy will consider that a type error since it's not  True. But it would be perfectly legal usage.